Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "BinaryBuilder"
 uuid = "12aac903-9f7c-5d81-afc2-d9565ea332ae"
 authors = ["Elliot Saba <staticfloat@gmail.com>"]
-version = "0.5.1"
+version = "0.5.2"
```

Comment:
Changing the manifest has no effect besides CI, what's the point of a new release?